Vagrantfile

```diff
@@ -12,15 +12,38 @@ Vagrant.configure('2') do |config|
   config.vm.define :test_docker do |node|
     node.vm.box = 'flavio/opensuse13-2'
     node.vm.box_check_update = true
+
     node.vm.provision 'shell', inline: <<EOS
+
+    # Install the latest zypper and docker.
     zypper ar http://download.opensuse.org/repositories/zypp:/Head/openSUSE_13.2/ zypp:Head
     zypper --gpg-auto-import-keys ref && zypper -n remove yast2-pkg-bindings libyui-ncurses-pkg6
     zypper --gpg-auto-import-keys ref && zypper -n install --from zypp:Head zypper libzypp
-    zypper --gpg-auto-import-keys ref && zypper -n install docker go
+    zypper --gpg-auto-import-keys ref && zypper -n install docker
+
+    # Since Go 1.4 is not in openSUSE 13.2 yet, let's use the one as provided
+    # by the Go project.
+    wget -q https://storage.googleapis.com/golang/go1.4.2.linux-amd64.tar.gz -O go.tar.gz
```

Comment:
why not installing it from http://download.opensuse.org/repositories/devel:/languages:/go/openSUSE_13.2/ ?